src/System.Data.SqlClient/src/Interop/ILocaleApiHelper.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Concurrent;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Data
+{
+    internal interface ILocaleApiHelper
```

Comment:
which philosophy that we follow ?
I don't think abstract will suit this. 
Interface will give cleaner solution for couple reasons 
- this is internal api which need to have exact similar functionality either from win api or mapper file.
- it will be switch based on whether it unix or unix.